### ISSUE TITLE
Momentum Threat Tracker

### ISSUE THREAD

**Issue by xanstin** [FabulistVtt/sta-lcars-ui#5]
May be a v11 issue but when you minimize it, it goes out of sight down and no way to find it other than reloading the page.

**Comment by deraviin** [comment]
Yeah it happens for me too, turning off the UI makes it possible to bring the tracker back but when the module is on it just slides completely off the screen never to be seen again.
